src/java.base/unix/classes/sun/nio/fs/UnixUserDefinedFileAttributeView.java

```diff
@@ -166,10 +166,10 @@ public int read(String name, ByteBuffer dst) throws IOException {
         assert (pos <= lim);
         int rem = (pos <= lim ? lim - pos : 0);
 
-        if (dst instanceof sun.nio.ch.DirectBuffer ddst) {
+        if (dst instanceof sun.nio.ch.DirectBuffer) {
```

Comment:
I assume this can be changed to test dst.isDirect() now and avoid the explicit cross package dependency.